scripts/push_sha_to_terraform.sh

```
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 ATLAS_ENVIRONMENT"
+  exit 1
+fi
+
+ATLAS_ENVIRONMENT=$1
+
+if [ -z "$ATLAS_TOKEN" ]; then
+  echo "Please set ATLAS_TOKEN variable in CircleCI project settings"
+  exit 1
+fi
+
+api_endpoint="https://atlas.hashicorp.com/api/v1"
+env_endpoint="${api_endpoint}/environments/${ATLAS_ENVIRONMENT}"
+
+# Push the new sha variable.
+curl ${env_endpoint}/variables \
```

Perhaps, but since we're failing early and explicitly showing commands (that's what `set -ex` does), if we get anything but `200` from the 1st `curl`, we'll fail the step. Same with the second step, and this one returns a JSON like `{"success": true}`. Mind you, these scripts are meant to abstract certain steps.